config/routes.rb

```diff
@@ -0,0 +1,51 @@
+Rails.application.routes.draw do
+  root 'posts#index'
+  # resources :users, only: [:new, :create]
+  get '/signup', to: 'users#new'
+  post '/signup', to: 'users#create'
+
+  resources :sessions, only: [:new, :create, :destroy]
```

Comment:
I read your comments and the cool part is I already gave you the solution. Somehow you missed it. You can remove that `resources :sessions` and you have to add `post '/signin', to: 'sessions#create'`. You can see ☝️ which I clearly mentioned. I already tested it and it works in my case without any error.